INFO 2023-10-06 16:41:48,290 autoreload D:\IOTAPI\Main_app\views.py changed, reloading.
INFO 2023-10-06 16:41:48,932 autoreload Watching for file changes with StatReloader
INFO 2023-10-06 16:41:53,297 autoreload D:\IOTAPI\Main_app\views.py changed, reloading.
INFO 2023-10-06 16:41:53,915 autoreload Watching for file changes with StatReloader
INFO 2023-10-06 16:43:16,536 autoreload D:\IOTAPI\Main_app\views.py changed, reloading.
INFO 2023-10-06 16:43:17,191 autoreload Watching for file changes with StatReloader
INFO 2023-10-06 16:44:02,813 autoreload D:\IOTAPI\Main_app\views.py changed, reloading.
INFO 2023-10-06 16:44:03,424 autoreload Watching for file changes with StatReloader
INFO 2023-10-06 16:44:36,956 autoreload D:\IOTAPI\Main_app\views.py changed, reloading.
INFO 2023-10-06 16:44:37,573 autoreload Watching for file changes with StatReloader
WARNING 2023-10-06 16:44:39,827 log Not Found: /favicon.ico
INFO 2023-10-06 16:45:37,841 autoreload D:\IOTAPI\Main_app\views.py changed, reloading.
INFO 2023-10-06 16:45:38,440 autoreload Watching for file changes with StatReloader
INFO 2023-10-06 16:46:23,045 autoreload D:\IOTAPI\Main_app\views.py changed, reloading.
INFO 2023-10-06 16:46:23,672 autoreload Watching for file changes with StatReloader
ERROR 2023-10-06 16:47:22,107 log Internal Server Error: /fecaf/logs
Traceback (most recent call last):
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\Main_app\views.py", line 79, in get
    return render(request, 'logs.html',{"data":text})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 518, in parse
    self.unclosed_block_tag(parse_until)
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 581, in unclosed_block_tag
    raise self.error(token, msg)
django.template.exceptions.TemplateSyntaxError: Unclosed tag on line 20: 'for'. Looking for one of: empty, endfor.
ERROR 2023-10-06 16:47:22,809 log Internal Server Error: /fecaf/logs
Traceback (most recent call last):
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\Main_app\views.py", line 79, in get
    return render(request, 'logs.html',{"data":text})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 518, in parse
    self.unclosed_block_tag(parse_until)
  File "D:\IOTAPI\ambiente_execucao\Lib\site-packages\django\template\base.py", line 581, in unclosed_block_tag
    raise self.error(token, msg)
django.template.exceptions.TemplateSyntaxError: Unclosed tag on line 20: 'for'. Looking for one of: empty, endfor.
WARNING 2023-10-06 16:47:22,922 log Not Found: /favicon.ico
INFO 2023-10-06 16:48:27,085 autoreload D:\IOTAPI\Main_app\views.py changed, reloading.
INFO 2023-10-06 16:48:27,753 autoreload Watching for file changes with StatReloader
WARNING 2023-10-06 18:49:02,083 log Not Found: /favicon.ico
WARNING 2023-10-06 18:50:02,368 log Not Found: /favicon.ico
WARNING 2023-10-06 19:03:37,232 log Not Found: /favicon.ico
INFO 2023-10-06 19:34:24,205 autoreload Watching for file changes with StatReloader
INFO 2023-10-06 19:37:23,789 autoreload Watching for file changes with StatReloader
WARNING 2023-10-06 19:39:17,943 log Not Found: /favicon.ico
WARNING 2023-10-06 19:40:41,040 log Not Found: /favicon.ico
WARNING 2023-10-06 19:42:04,242 log Not Found: /favicon.ico
WARNING 2023-10-06 19:42:07,626 log Not Found: /favicon.ico
INFO 2023-10-19 22:53:08,181 autoreload Watching for file changes with StatReloader
INFO 2023-10-19 22:53:24,284 autoreload Watching for file changes with StatReloader
WARNING 2023-10-19 22:53:36,673 log Not Found: /
WARNING 2023-10-19 22:53:36,753 log Not Found: /favicon.ico
WARNING 2023-10-19 22:58:40,409 log Not Found: /favicon.ico
WARNING 2023-10-19 22:58:59,407 log Not Found: /favicon.ico
WARNING 2023-10-19 22:59:21,679 log Not Found: /favicon.ico
WARNING 2023-10-19 22:59:55,026 log Not Found: /favicon.ico
WARNING 2023-10-19 23:00:13,384 log Not Found: /favicon.ico
WARNING 2023-10-19 23:02:51,326 log Not Found: /favicon.ico
WARNING 2023-10-19 23:04:39,791 log Not Found: /favicon.ico
INFO 2023-10-19 23:05:54,304 autoreload Watching for file changes with StatReloader
WARNING 2023-10-19 23:06:39,991 log Not Found: /favicon.ico
